### PR TITLE
Automated chart bump jenkins-2.5.4

### PR DESCRIPTION
**Pull request by d2iq-dispatch**

### CODE REVIEW

**Comment by cprovencher** [comment]
Dispatch failing because of resource issues I believe

File: addons/jenkins/1.x/jenkins.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 metadata:
   name: jenkins
   labels:
     kubeaddons.mesosphere.io/name: jenkins
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-2"
     appversion.kubeaddons.mesosphere.io/jenkins: "1.9.4"
-    values.chart.helm.kubeaddons.mesosphere.io/jenkins: "https://raw.githubusercontent.com/helm/charts/master/stable/jenkins/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/jenkins: "https://raw.githubusercontent.com/helm/charts/6ec6034/stable/jenkins/values.yaml"
```
Comment (takirala) [inline addons/jenkins/1.x/jenkins.yaml:10]: how about using 
https://raw.githubusercontent.com/jenkinsci/helm-charts/main/charts/jenkins/values.yaml to use the values from `jenkinsci` org.

(OR https://raw.githubusercontent.com/jenkinsci/helm-charts/f251fce11b8c7dd376059732d965112e505907a3/charts/jenkins/values.yaml to pin it)

**Review by takirala — APPROVED** [review]
Looks good, i think we can just use the `jenkinsci/helm-charts` for values.yaml as well

**Review by cprovencher — DISMISSED** [review]

**Comment by shaneutt** [comment]
another automated PR took care of this :+1:

**Pull request closed without merging**